CrashReporting/Sources/Infraestructure/DefaultLogsBuilder.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  DefaultLogsBuilder.swift
+//  
+//
+//  Created by Martin Dutra on 28/6/22.
+//
+
+import Foundation
+import Logger
+
+class DefaultLogsBuilder: LogsBuilder {
```

Comment:
Maybe it makes sense to name this something like `GoBotLogsBuilder` since it is tightly coupled to where the GoBot stores the logs on disk?